package.json

```diff
@@ -44,6 +44,7 @@
     "lerna": "^5.5.0",
     "nock": "^13.2.9",
     "npm-run-all": "^4.1.5",
+    "nx": "^16.5.1",
```

Comment:
Does this need an _nx.json_ file now too?

Reply:
Yep - the commit fails if that's not present